### DISCUSSION TITLE
Help with issue "Problem opening input file .win" on running vasp_ncl

### DISCUSSION THREAD

**Discussion by MJAJ333** [Chengcheng-Xiao/VASP2WAN90_v2_fix#21] [Q&A]
Hello all/ developer!

Please could someone advise with this issue, as I am unable to find a solution.

My cluster has followed the steps from this page and made custom compilation using this version of VASP and applied the patch, however we are running into this issue early on in the running process.

I assume it is an issue with the patch, version, interface, or the cluster, which for some reason doesn't seem to read the INCAR and Wannier90.win properly. I get the error " Error: Problem opening input file .win" in the std.out, but I can't get any other info. I have followed the given example from the tutorial and ran on a single node with no MPI settings but I cannot get it to read the inputs successfully. It is reading the INCAR partially as it recognizes a setting (collinear, or if there is spin or no spin present) but then stops. I also tried with no Wannier90.win file so it can generate its own, but it only prints one line (about the number of bands) and stops so I assume there is an issue with the interface. 
The output text in the std.out is shown below.

 *---------------------------------------*
             VASP2WANNIER90            
 *---------------------------------------*
           Non-collinear spin           
 Error: Problem opening input file .win
Error: examine the output/error file for details

---
Is the issue maybe what version of Wannier90 is being used? I think maybe in this build it is using wannier90/3.1.0. What version of Wannier90 was used in the making of this patch?

Thank you!
Dr. Malcolm Jardine, Carnegie Mellon University

**Comment by Chengcheng-Xiao** [comment]
Hi Malcolm, have you tried naming the input file for Wannier90 `wannier90.win`? When it comes to file names, Linux is usually case-sensitive.

**Reply by MJAJ333** [reply]
Thank you for the reply! 

The file that is there is called "wannier90.win" yes. I have done an example where I copied and pasted the scf and wannier-gen files from your example, and checking again the input file for Wannier90 does have the correct name.

Also, if I don't provide the wannier90.win file VASP does start to make a wannier90.win, but only gets ones line in and then stops

**Reply by Chengcheng-Xiao** [reply]
Interesting, I have never encountered such an issue. 
Please make sure you are using __VASP.5.4.4.pl2__ with __Wannier90 v3.1.0__.

>  I have done an example

Does the example run successfully? 

> Error: Problem opening input file .win

This error comes from subroutine `param_in_file` in `param_read`, called by `wannier_setup` that is, in turn, called by VASP2WANNIER90 interface (`mlwf.F`).
Specifically, it's
```fortran
101 call io_error('Error: Problem opening input file '//trim(seedname)//'.win')
```
Since in your error, no seed name was printed out, I suspect VASP is not passing seedname correctly to the interface.

I have no way of checking how this happened without having access to the source files and the compilers you used. So I would suggest you check with your sys-admin first and make sure they are using exactly the same version of VASP and wannier90 and are following my steps.

**Reply by MJAJ333** [reply]
Thanks for the info! I have taken this and I am now speaking with the sys-admin who did the compilation, we will check we are using the correct versions as you suggested, and see if we can work out the seed name issue.
I will get back to you when they give me more information, and maybe give more info on the build if necessary.

> Does the example run successfully?

The example (example one with Bi) runs correctly until the wannier90 interface is called, I have not plotted the outputs but the scf calculation using the VASP.5.4.4.pl2  build seems to run normally. 

By seedname, do you mean the wannier90.win file that is printed out? Because if I don't have a wanner90.win file initially, one is made.

Thanks for the help!
Malcolm

**Reply by Chengcheng-Xiao** [reply]
By seedname, I mean "wannier90". Unlike QuantumEspresso, in VASP, the seed name is hard-coded and will always be "wannier90". I said "no seed name was printed out" because if the seed name is read in successfully, that specific error message will be:
```
Error: Problem opening input file wannier90.win
```
And since it's:
```
Error: Problem opening input file .win
```
The seed name is not correctly passed into the function `param_in_file`.

**Reply by MJAJ333** [reply]
I see, thank you very much for explanation. I will see what me my cluster administrators can do and get back to you!
I would very much like to see this work so I can see the spin character of my GW band structure
